docs/content/introduction/faq.md

```diff
@@ -0,0 +1,43 @@
+# Veelgestelde vragen
+
+## Inhoud
+
+* Wie is de opdrachtgever?
+* Wat is de opdacht?
+* Wat gaat er concreet gemaakt worden?
+* Wat betekent dit voor leveranciers?
+* Wat betekent dit voor gemeentes?
+* Worden zaaksystemen hiermee beter, goedkoper, sneller, beter uitwisselbaar, etc.?
+* Is de referentie-implementatie bruikbaar in productie?
+* Komt er een landelijke versie van dit "product"?
+
+
+## Wie is de opdrachtgever?
+
+TODO: Een VNG taskforce
+
+## Wat is de opdacht?
+
+TODO: Maak een nieuwe standaard
+
+## Wat gaat er concreet gemaakt worden?
+
+TODO: Zie [realisatie](https://github.com/VNG-Realisatie/gemma-zaken/blob/master/docs/content/introduction/productvisie.md#realisatie)
+
+## Wat betekent dit voor leveranciers?
+
+TODO: Leveranciers kunnen de nieuwe specificatie implementeren, testen
+
+## Wat betekent dit voor gemeentes?
+
+
+## Worden zaaksystemen hiermee beter, goedkoper, sneller, beter uitwisselbaar, etc.?
```

Comment:
In GEMMA 2 kennen we het Zaaksysteem niet meer. Maar begrijpt iedereen het concept ZaakRegistratieComponent?

Reply:
Ik zie graag je PR tegemoet :)